frontend/src/Setting/Admin/PassWord/PassWordPane.tsx

```diff
@@ -16,6 +16,20 @@ type Props = {
 function PasswordPane({ visible, setVisible }: Props) {
   const [password, setPassword] = useState("");
 
+  function pressEnter(event: React.KeyboardEvent<HTMLElement>) {
+    if (event.key === "Enter") {
+      passwordInputted();
```

Comment:
```suggestion
      authPassword();
```
メソッド名は`authPassword()`とかの方がいい気がします

Reply:
メソッド名変更は同意．

passwordを目的語にするなら動詞はverifyがいいと思います
* verify：検証．O（目的語）が正しいことを確かめる
* authenticate：認証．Oが本物であることを確かめる
* authorize：認可．Oに～する許可を与える，Oする許可を与える